src/main/java/teammates/common/datatransfer/FeedbackSessionAttributes.java

```diff
@@ -316,6 +316,24 @@ public boolean isClosingWithinTimeLimit(int hours) {
                && differenceBetweenDeadlineAndNow >= hours - 1
                && differenceBetweenDeadlineAndNow < hours;
     }
+    
+    public boolean isClosedWithinPastHour() {
```

Comment:
Add header comments to the method. Need to specify subtleties of the method behavior, such as the precision level. i.e. at which exact time the method goes from false to true?